### DISCUSSION TITLE
Slicing special files - SS does not retain internal structure

### DISCUSSION THREAD

**Discussion by Alan5596** [supermerill/SuperSlicer#788] [Q&A]
First, thanks for the SS - I have recently started using it in preference to S3D which I have used until I got my Voron up and running.  I'm not a coding guy so so not understand the complexities of slicing etc, but I have been trying to use SS to print some R/C model files from 3DlabPrint  and SS does not retain the internal structure after slicing.  

I have trawled a few R/C site and here is some info that might explain it to you.

"SuperSlicer is using algorithms that are based on manifold body analysis.
It needs to know what is "inside" and what is "outside" of a body.  That is why if the faces are inverted it can't print - and if there is single face it will not recognize it.  3D lab print models have internal surfaces "inverted" - and thus they are non recognized by Slic3r, PrusaSlicer, SuperSlicer programs. 

I'm not sure if there is a way around it but if anyone is interested you can try with a sample part from here...

https://www.thingiverse.com/thing:1027329

**Comment by supermerill** [comment]
imho, It seems like a bad stl file.
You can try to make it manifolded it with meshmixer.
You can also open an issue in prusaslicer (as it slices the same in prusaslicer and they made some fixes to better slices bad stl)..

**Comment by Alan5596** [comment]
Thanks, they can't be bad Stl files as I have printed several models using S3D and the parts are perfect. The problem is S3D outputs a lot of stuff that I have not been able to sort out to use the Gcode to print on the Voron, otherwise I would happily carry on using S3D for those files and SS for everything else.  Maybe I will just keep going at editing an S3D Gcode file until the Voron doesn't reject any lines of code.

**Reply by neophyl** [reply]
Isnt this the same as discussed over here for Prusa Slicer ?  https://github.com/prusa3d/PrusaSlicer/issues/3062

**Reply by Jiter** [reply]
PrusaSlicer added a Slicing Mode to Slice this files:
https://github.com/prusa3d/PrusaSlicer/commit/20ba7c0a1f934430ecdd300bb25f2377e552cc38

Is there any possibility that we can have this to in SS?

**Reply by neophyl** [reply]
Jiter as far as I know the current plan is to start merging in against Prusa Slicer 2.4 after its had a proper release.  No point until then as I imagine there will be a significant amount of work to get it to merge and not break something in SuSi.
So all the new goodies should make it across.
